### ISSUE TITLE
CTRL+Click not working

### ISSUE THREAD

**Issue by vincenzovitale** [wayou/vscode-todo-highlight#92]
When I try to follow the link I get this message:
![image](https://user-images.githubusercontent.com/4376527/39048705-a8f469f4-449e-11e8-9c7f-4a35f5d928a9.png)

**Comment by wayou** [comment]
have you tried `todohighlight.toggleURI: true`?

**Comment by changeling** [comment]
I got caught by this as well. Perhaps adding a line to the NOTICE at the top of the Extension info page regarding this? Such a simple fix, but my initial RTFMWOPA (Read [Past tense: rĕd,  /ɹɛd/] The Fine Manual Without Paying Close Attention) missed this:

Something like:

---
NOTICE

Many report that the List hilighted annotations command not working, make sure you have the file types included via todohighlight.include.

If you receive an `Unable to open <filename>: File not found` error with `<control>-` or `<command>-<click>  <filepath>`, ensure that you've set `todohighlight.toggleURI` to either `true` or `false`, depending on your environment.
---

(Thanks for this extension. It's now  indispensable to my workflow.)

**Comment by wayou** [comment]
> "Alt + click to follow link" doesn't work

it seems that both uri pattern dont work on your side.
actually I have no idea how to handle this cause it's related to the vscode itself.
but once I have a Ubuntu vm installed I will look into this.

**Comment by johnwatsondev** [comment]
@wayou Yes. You are right. It worked perfectly when I disabled the [Vim Plugin](https://github.com/VSCodeVim/Vim).
